kibana/templates/configmap-dashboardimport.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.dashboardImport.enabled }}
```

Comment:
Changes to this configmap will not cause the container to be redeployed. https://helm.sh/docs/charts_tips_and_tricks/#automatically-roll-deployments-when-configmaps-or-secrets-change